pfio/v2/fs.py

```diff
@@ -338,25 +335,28 @@ def from_url(url: str, **kwargs) -> 'FS':
 
         create (bool): Create the specified path doesn't exist.
 
-        reset_on_fork (bool): Reset stateful objects (e.g. connection
-            to the remote system) after fork.
-
     .. note:: Some FS resouces won't be closed when using this
         functionality.
 
     .. note:: Pickling the FS object may or may not work correctly
         depending on the implementation.
 
     '''
-    parsed = urlparse(url)
-    force_type = kwargs.pop('force_type', None)
+    if kwargs.pop('reset_on_fork', None) is not None:
+        warnings.warn(
+            "reset_on_fork is deprecated. PFIO resets on fork by default",
+            category=DeprecationWarning,
+            stacklevel=2
```

Comment:
By configuring stacklevel to two, user can see the stacktrace of caller (in user's training script) instead of where warning happen (in PFIO).